src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
```

Comment:
There is a variable `$days-gap` used here, but it is not defined in the provided code. Ensure that `$days-gap` is defined in the imported `variables` file or elsewhere in your code.